outbound.js

```diff
@@ -352,7 +352,10 @@ exports.send_email = function () {
     var from = arguments[0],
         to   = arguments[1],
         contents = arguments[2];
-        var next = arguments[3];
+    var next = arguments[3];
+    var options = arguments[4];
+
+    var dot_stuffing = ((options && typeof(options.dot_stuffing) !== 'undefined') ? options.dot_stuffing : true);
```

Comment:
I've renamed it to "dot_stuffed" and negated the logic (default ```false```), so it describes the passed ```contents```.